### ISSUE TITLE
More information of Upgrade guide for 6.x

### ISSUE THREAD

**Issue by terapyon** [plone/volto#1612]
We finished an upgrade from 4.6 to 6.2 for our custom volto app.
We found no documentation for the upgrade guide for details. It's two important files.

- config.js
- start-client.jsx

config.js needed to add 2 lines.
```
export const addonRoutes = [];
export const addonReducers = {};
```

start-client.jsx needed modify `loadableReady` wraps.

Please add the upgrade guide, if my suggestion is true.

**Comment by tisto** [comment]
@terapyon we are a bit short on time right now. Could you imagine to create a Pull Request and add this to the upgrade guide?

**Comment by tiberiuichim** [comment]
@terapyon Indeed, these need to be added in config.js:

```
export const addonRoutes = [];
export const addonReducers = {};
```

**Comment by tisto** [comment]
@terapyon @jackahl and I investigated this and indeed https://github.com/plone/volto/pull/1542/files introduced a breaking change that I missed. We will update the Volto 6.x.x upgrade guide even though this breaking change was introduced in Volto 6.1.0. Re-releasing 6.1.0 as 7.0.0 is not feasible any longer since we already released 6.2.x and 6.3.x releases. Sorry for the inconvenience caused by this. We should have seen this before merging/releasing.
